src/iomap.cpp

```diff
@@ -198,9 +198,22 @@ bool IOMap::loadMap(Map* map, const std::string& identifier)
 				setLastErrorString("Invalid map node.");
 				return false;
 			}
-
-			uint16_t base_x = area_coord.x;
-			uint16_t base_y = area_coord.y;
+			
+			uint16_t base_x = 0;
```

Comment:
no it no breaks. It is just need, becouse declarations must be assigned to anything. Default have 0.
In down code I am have "else", and that assignet to base_x = area_coord.x; (it be standard code).